week-1-sean-wallace.js

```diff
@@ -0,0 +1,208 @@
+//const temp3 equals () and returns "212 is at boilin point" console log the temp3
+
+const temp1 = () => {
+    return "42 is below boiling point"
+}
+console.log(temp1())
+const temp2 = () => {
+     return "350 is above boiling point" 
+}
+console.log(temp2())
+
+const temp3 = () => {
+  return "212 is at boiling point"  
+}
+console.log(temp3())
+
+// --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
+
+// Pseudo code: let myarray equal (myNumbers1+myNumbers2) then console log (myArray and the .length of the array)
+
+
+const myNumbers1 = [3, 7, 0, 36, -9]
+const myNumbers2 = [8, -7, 42, 9, 13]
+
+let myArray = (myNumbers1.concat(myNumbers2))
+console.log(myArray.length)
+
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
+
+// Pseudo code: Use let string of const currentCohort ("Delta 2022") then string = the string and .reverse("Delta 2022) then .join("") console log (string) 
+
+const currentCohort = "Delta 2022"
+let string = ("Delta 2022")
+string = [...string].reverse().join("");
+
+console.log(string);
+
+
+
+
+// --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+
+// Pseudo code: Use for, if, else, for let index = 0; then i < myArray and .length, i++ for the value before the increment. if myArray index modulo by 2 strict inequal the 0 console.log odd else if myArray i console.log even.
+
+const myArray = [13, 34, 5, 10, 27, 42]
+for(let i = 0; i < myArray.length; i++) 
+   if(myArray[i] % 2 !== 0) {
+      console.log(`odd`)  
+    }
+    else if(myArray[i]) {
+        console.log(`even`)
+      }
+    
+
+
+
+
+
+// --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
+
+// Pseudo code: console.log number2-number1; console.log number3-number4
+
+// Set one:
+
+const number1 = 58
+const number2 = 100
+console.log(number2-number1)
+
+// Set two:
+const number3 = 27
+const number4 = 24
+console.log(number3-number4)
+
+// ASSESSMENT 1: Coding Conceptual Questions
+
+// Examine the following examples.
+
+// First, predict the outcome based on your understanding of the code.
+// Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
+
+// --------------------INSTRUCTOR EXAMPLE: What will this log?
+const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+//console.log(colors.push("indigo"))
+
+// a) Your answer: will push indigo to the end
+// b) Verify and explain: .push places any variable to the end of a string.
```

Comment:
Make sure you _verify_ the answer by uncommenting the console log on line 88.  The return here is actually 5.